packages/core/src/chains.ts

```diff
@@ -1,14 +1,22 @@
 export {
   arbitrum,
   arbitrumGoerli,
+  avalanche,
+  avalancheFuji,
+  bsc,
+  bscTestnet,
+  fantom,
+  fantomTestnet,
   foundry,
   goerli,
   hardhat,
-  sepolia,
   localhost,
   mainnet,
   optimism,
   optimismGoerli,
   polygon,
   polygonMumbai,
-} from './constants/chains'
+  sepolia,
+} from '@wagmi/chains'
```

Comment:
Curious what you think about exporting everything from `@wagmi/chains` by default? Might make it easier to bump new versions and we will have the snapshot test as a way to review changes.

Reply:
Yeah, makes sense to me!